README.md

```diff
@@ -2,20 +2,30 @@
 
 ## Cromwell run
 
-Import summary stats using [import.wdl](wdl/import.wdl). Prepare a list of summary stats like in reference configuration file (one summary stat bucket path per line) [import.json](wdl/import.json).
+Import summary stats using [import.wdl](wdl/import.gwas_pipeline.wdl). Prepare a list of summary stats like in reference configuration file (one summary stat bucket path per line) [import.json](wdl/import.gwas_pipeline.json).
 
 After successful import run, copy generated file to a single bucket using proper file structure using [copy_cromwell_import_to_bucket_puddle.py ](scripts/copy_cromwell_import_to_bucket_puddle.py).
 
-Parameters needed are cromwell hash and path to destination bucket.
-copy_cromwell_import_to_bucket_puddle.py cromwell_hash gs://bucket_for_deployment_pickerupper/v8/
+Parameters needed are cromwell hash and path to destination bucket: `copy_cromwell_import_to_bucket_puddle.py cromwell_hash gs://bucket_for_deployment_pickerupper/v8/`
```

Comment:
when using standard import we still have to create the phenotype json externally.
You could add step here how to generate that. Juha K has been doing that semi-manually but would be great to have all steps here.

Reply:
Ah right, somehow I assumed the phenotype json was just for the meta results. I'll add proper info about that process.

Reply:
Added example how to create the phenotype json from R7 with the help of the custom json script. See https://github.com/FINNGEN/pheweb/pull/143/commits/9e4d02a4cdff0989a5f676808ce4febe505a43e6